### PR TITLE
Release uploading should include kyuubi-spark-connector-hive

### PR DESCRIPTION
**Pull request by pan3793**
### _Why are the changes needed?_

The module is missed because it's only included when `spark-3.3` is activated.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: build/release/release.sh [hunk 1]
```diff
@@ -94,17 +94,14 @@ upload_svn_staging() {
 }
 
 upload_nexus_staging() {
-  ${KYUUBI_DIR}/build/mvn clean deploy -DskipTests -Papache-release,flink-provided,spark-provided,hive-provided \
-    -s "${KYUUBI_DIR}/build/release/asf-settings.xml"
   ${KYUUBI_DIR}/build/mvn clean deploy -DskipTests -Papache-release,flink-provided,spark-provided,hive-provided,spark-3.1 \
     -s "${KYUUBI_DIR}/build/release/asf-settings.xml" \
     -pl extensions/spark/kyuubi-extension-spark-3-1 -am
```
Comment (ulysses-you) [inline build/release/release.sh:99]: shall we also cleanup this line ?

File: build/release/release.sh [hunk 2]
```diff
@@ -94,17 +94,14 @@ upload_svn_staging() {
...
   ${KYUUBI_DIR}/build/mvn clean deploy -DskipTests -Papache-release,flink-provided,spark-provided,hive-provided,spark-3.2 \
     -s "${KYUUBI_DIR}/build/release/asf-settings.xml" \
     -pl extensions/spark/kyuubi-extension-spark-3-2 -am
```
Comment (ulysses-you) [inline build/release/release.sh:102]: ditto

**Review by ulysses-you — APPROVED** [review]

File: build/release/release.sh (hunk 1 above)
Comment (pan3793) [reply build/release/release.sh:99]: it's by design, we only need to deploy `kyuubi-extension-spark-3-1` for spark-3.1

**Comment by pan3793** [comment]
Thanks, merged to master/1.7

**Pull request closed without merging**